### PR TITLE
{bio}[intel/2016a] i-cisTarget v20160602, STAMP v1.2, Cluster-Buster v20160106, ...

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/241d6f042db137b9852ef1564b3873ec for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2626.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b97816f7702c81c442844c00caec026d for a full test report.

File: easybuild/easyconfigs/k/Kent_tools/Kent_tools-20130806-linux.x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+sources = [SOURCE_TAR_GZ]
+
+postinstallcmds = [
+    "cp -a %(builddir)s/Kent_tools/blat/{blat,gfClient,gfServer} %(installdir)s/bin",
+    "cp -a %(builddir)s/Kent_tools/blat/FOOTER.txt %(installdir)s/bin/FOOTER_blat.txt",
+]
+
+sanity_check_paths = {
+    'files': ['bin/FOOTER', 'bin/blat', 'bin/getRna', 'bin/liftOver', 'bin/mafGene', 'bin/splitFile', 'bin/twoBitToFa'],
+    'dirs': [],
```
Comment (pescobar) [inline easybuild/easyconfigs/k/Kent_tools/Kent_tools-20130806-linux.x86_64.eb]: cosmetics....

`'files': ['bin/%s' % x for x in ["aaa", "bbbb", "cccc"]]`

Comment (boegel) [inline easybuild/easyconfigs/k/Kent_tools/Kent_tools-20130806-linux.x86_64.eb]: OK, will change, thanks

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8beca38906c7a8d7e72b0d40b511057b for a full test report.

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Thanks for the review @pescobar & @wpoely86!

**Pull request merged**